tests/e2e/test_rosacli_cluster.go

```diff
@@ -3306,6 +3314,52 @@ var _ = Describe("HCP cluster creation supplemental testing",
 				clusterID = clusterList.ClusterByName(testingClusterName).ID
 				Expect(clusterID).ToNot(BeNil())
 			})
+
+		It("Check single AZ hosted cluster can be created - [id:54413]",
+			labels.Critical, labels.Runtime.Day1Supplemental,
+			func() {
+				testingClusterName = common.GenerateRandomName("c54413", 2)
+				flags, err := profilehandler.GenerateClusterCreateFlags(customProfile, rosaClient)
+				Expect(err).ToNot(HaveOccurred())
+
+				command := "rosa create cluster --cluster-name " + testingClusterName + " " + strings.Join(flags, " ")
+				rosalCommand := config.GenerateCommand(command)
+				if rosalCommand.CheckFlagExist("--multi-az") {
+					rosalCommand.DeleteFlag("--multi-az", false)
+				}
+				if rosalCommand.CheckFlagExist("--multi-az") {
```

Comment:
Could you PTAL, you have written the same code twice? Thanks

Reply:
removed second occurrence